legal-api/report-templates/specialResolutionCorrectionApplication.html

```diff
@@ -27,6 +27,8 @@
          </div>
       </div>
       [[correction/businessDetails.html]]
+      [[correction/addresses.html]]
+      [[correction/directors.html]]
       <div class="container">
          [[special-resolution/resolutionApplicationCorrection.html]]
```

Comment:
This file should be removed.  Correction.html already covers this.

Reply:
Yup, can be captured in another file in `correction/` or included in `businessDetails.html`

Reply:
removed